portal/src/graphql/adminapi/AuditLogScreen.tsx

```diff
@@ -12,23 +12,20 @@ import {
 } from "react-router-dom";
 import {
   ICommandBarItemProps,
-  IDropdownOption,
   addDays,
```

Comment:
Investigate whether it is possible to let the list to take available height in mobile view. The pagination control should be visible without scrolling. Apply this to other list screen as well, such as the UsersScreen.

Reply:
wontfix